app/assets/javascripts/deleteComment.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+  $('.commentDelete').click(function(event) {
+    event.preventDefault();
+    var meetup = this.parentElement.parentElement.parentElement.parentElement
+    var meetup_id = meetup.id
+    commentId = this.parentElement.id.replace("comment","")
+    $.ajax({
+      url: "/meetups/" + meetup_id + "/comments/" + commentId,
```

Comment:
Mixed double and single quotes.<br>Identifier 'meetup_id' is not in camel case.<br>'commentId' is not defined.